adapters/ddf-adapter/pom.xml

```diff
@@ -55,6 +54,26 @@
           <groupId>net.sf.ehcache</groupId>
           <artifactId>ehcache</artifactId>
         </exclusion>
+        <exclusion>
```

Comment:
❓ Would it make sense to declare all these exclusions only in the top-level parent pom to avoid having to repeat those everywhere else?

Question applies to all other artifacts here with exclusions

Reply:
These dependencies and exclusions here only show up for the ddf-adapter that has the ddf/cxf dependencies. I think they make sense here for now.